src/components/layout/Shell.tsx

```diff
@@ -42,39 +45,38 @@ export function Shell(props: Props) {
           >
             <Image src="/metabase-logo-with-wordmark.svg" />
 
-            <Box className="md:hidden">
+            <Flex display={{ base: "flex", sm: "none" }}>
```

Comment:
This was quite confusing when changing from `md` to `sm`, because we changed from tailwind breakpoint to Mantine breakpoint, but they have the same value.

I wonder if we really need to wrap the burger menu inside a flex since it's the only child. And seems like we can pass `display` prop to `Burger` directly since it's also a Mantine component.

I tried with this and it seems to work (without `Flex`). This simplifies markup a bit.

```ts
<Burger
  display={{ sm: "none" }}
  opened={isMobileNavOpen}
  onClick={toggleMobileNav}
  aria-label="Toggle navigation"
  color="#eee"
/>
```

Reply:
@WiNloSt Tailwind breakpoint is actually slightly different than Mantine's breakpoint, so there are subtle things that break on specific viewport sizes if we use Tailwind's breakpoints. Let me try out your markup instead, thanks!

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/a60d57c11fc38ada2df06eb7a368e382cff0e408